aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -384,6 +388,17 @@ module aptos_framework::multisig_account {
         (voted, vote)
     }
 
+    #[view]
+    public fun available_transaction_queue_capacity(multisig_account: address): u64 acquires MultisigAccount {
+        let multisig_account_resource = borrow_global_mut<MultisigAccount>(multisig_account);
+        let num_pending_transactions = multisig_account_resource.next_sequence_number - multisig_account_resource.last_executed_sequence_number - 1;
```

Comment:
I think this could underflow for `last_executed_sequence_number` = 0, which would happen for a new multisig account

Unfortunately that will be confusing for new users:

1. Someone creates a new multisig account
2. They call `available_transaction_queue_capacity`
3. The view function aborts

Reply:
`next_sequence_number` is initially 0. So, there should be no underflow: https://github.com/aptos-labs/aptos-core/blob/775cbad4c3144cd55b81ceaba5b7fe1bf92a1d0b/aptos-move/framework/aptos-framework/sources/multisig_account.move#L597-L597